### PR TITLE
Added recipe for versiontools

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/versiontools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - versiontools.setuptools_hooks
+    - versiontools.bzr_support
+    - versiontools.git_support
+    - versiontools.hg_support
+
+about:
+  home: https://launchpad.net/versiontools
+  license: LGPL 3.0
+  # license_file: No LICENSE included in package
+  license_family: Public-Domain
```
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: Should be LGPL then, right?

File: recipes/versiontools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    - versiontools
+    - versiontools.setuptools_hooks
+    - versiontools.bzr_support
+    - versiontools.git_support
+    - versiontools.hg_support
+
+about:
+  home: https://launchpad.net/versiontools
+  license: LGPL 3.0
+  # license_file: No LICENSE included in package
```
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: Could you please add a PR upstream to add this to `MANIFEST.in`?

File: recipes/versiontools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    # - setuptools
```
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: Can we drop this then?

File: recipes/versiontools/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: ...or perhaps GPL then? Not sure how the license families work. @goanpeca, could you please weight in here?

Comment (goanpeca) [inline recipes/versiontools/meta.yaml]: Inam not sure either LGPL or GPL?

@msarahan, thoughts?

File: recipes/versiontools/meta.yaml (hunk 2 above)
Comment (pmlandwehr) [inline recipes/versiontools/meta.yaml]: Submitted an issue - see [issue 3](https://github.com/zyga/versiontools/issues/3)

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

File: recipes/versiontools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    - versiontools.setuptools_hooks
+    - versiontools.bzr_support
+    - versiontools.git_support
+    - versiontools.hg_support
+
+about:
+  home: https://launchpad.net/versiontools
+  license: LGPL 3.0
+  # license_file: No LICENSE included in package - see https://github.com/zyga/versiontools/issues/3
+  license_family: Public-Domain
```
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: Should be LGPL.

File: recipes/versiontools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    - versiontools
+    - versiontools.setuptools_hooks
+    - versiontools.bzr_support
+    - versiontools.git_support
+    - versiontools.hg_support
+
+about:
+  home: https://launchpad.net/versiontools
+  license: LGPL 3.0
+  # license_file: No LICENSE included in package - see https://github.com/zyga/versiontools/issues/3
```
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: xref: https://github.com/zyga/versiontools/issues/3

File: recipes/versiontools/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  preserve_egg_dir: True
+
```
Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: nit: Please drop extra newline.

Comment (jakirkham) [inline recipes/versiontools/meta.yaml]: Please address at the feedstock.

**Pull request merged**